### PR TITLE
Use Insurely data for pre-selecting insurance

### PR DESCRIPTION
**Pull request by simonauner**
## What?

Pre-select Car tier based on previous insurance (from Insurely).

Use `insuranceProviderDisplayName` instead of hard-coded insurance company names on Offer page.

## Why?

**Ticket(s): []**

### CODE REVIEW

**Comment by robinandeer** [comment]
Current dependencies on/for this PR:
* master
  * **PR #1179** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/web-onboarding/1179" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/web-onboarding/1179?utm_source=stack-comment).

File: src/client/pages/Offer/index.tsx [hunk 1]
```diff
@@ -119,9 +120,19 @@ export const OfferPage = ({
     `${localePathPattern}/new-member/sign/${quoteCartId}`,
   )
 
+  const dataCollectionId =
```
Comment (alebedev) [inline src/client/pages/Offer/index.tsx:123]: This is quite long selector, does it make sense to extract it?

File: src/client/pages/Offer/utils.ts [hunk 2]
```diff
@@ -453,6 +455,15 @@ export const getBundleVariantFromInsuranceTypesWithFallback = (
 
   if (matchingVariant) return matchingVariant
 
+  const matchingPreviousInsurance = variants.find(
```
Comment (alebedev) [inline src/client/pages/Offer/utils.ts:458]: Same logic as in InsuranceSelector, does it make sense to extract it?

**Review by alebedev — APPROVED** [review]
Looks good

File: src/client/pages/Offer/utils.ts (hunk 2 above)
Comment (robinandeer) [reply src/client/pages/Offer/utils.ts:458]: Let me address that in a follow-up PR since we also need to match on exposure :)

**Pull request merged**